packages/shopify-app-remix/src/server/authenticate/admin/helpers/redirect-to-bounce-page.ts

```diff
@@ -7,13 +7,17 @@ export const redirectToBouncePage = (params: BasicParams, url: URL): never => {
 
   // Make sure we always point to the configured app URL so it also works behind reverse proxies (that alter the Host
   // header).
-  url.searchParams.set(
+  const searchParams = new URLSearchParams(url.search);
+  searchParams.delete('id_token');
```

Comment:
It's because the `id_token` in the parameter can be invalid and given that we are redirecting to the bounce page to get a session token, it's redundant and also erroneous to pass along the `id_token` (because the we would end up trying to validate `id_token`).